apps/api/src/routes/applicationRouter.ts

```diff
@@ -339,4 +341,45 @@ applicationRouter.post(
 	),
 );
 
+// Endpoint for reps to request revisions
+applicationRouter.post(
+	'/request-revisions',
+	jsonParser,
+	withBodySchemaValidation(applicationRevisionRequestSchema, apiZodErrorMapping, async (req, res) => {
+		const { applicationId, revisionData, role } = req.body;
+
+		if (!role || (role !== 'INSTITUTIONAL_REP' && role !== 'DAC_MEMBER')) {
+			res.status(400).json({ message: 'Invalid request: Invalid role' });
+		}
```

Comment:
Please separate into two endpoints, `/dac/request-revisions` and `/rep/request-revisions`
And then this `role !== 'INSTITUTIONAL_REP'` check should just check for one role per end point